src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/OverlayList.js

```diff
@@ -0,0 +1,149 @@
+// @flow
+import Portal from 'react-portal';
+import {observer} from 'mobx-react';
+import React from 'react';
+import {action, computed, observable} from 'mobx';
+import Backdrop from '../Backdrop';
+import Option from './Option';
+import overlayListStyles from './overlayList.scss';
+import type {OverlayListDimensions} from './types';
+import OverlayListPositioner from './OverlayListPositioner';
+
+@observer
+export default class OverlayList extends React.PureComponent {
+    props: {
+        isOpen: boolean,
+        children?: React.Element<*>,
+        onRequestClose?: () => void,
+        /** The top coordinate relative to which the list will be positioned **/
```

Comment:
Styleguidist only inlcudes comments beginning with a double asterix into its documentation. Even if this component doesn't get a designated docu section, I'd suggest keeping the comment format for all components consistent over the project.